packages/forma-36-react-components/package.json

```diff
@@ -30,14 +30,14 @@
     "@storybook/addon-essentials": "^6.0.21",
     "@storybook/addon-knobs": "^6.0.21",
     "@storybook/addon-storysource": "^6.0.21",
-    "@storybook/react": "6.1.0-alpha.10",
+    "@storybook/react": "6.0.27",
```

Comment:
Why the downgrade?

Reply:
I think when I was implementing popper.js for the first time I had to upgrade to the alpha version for some reason

And then now I downgraded to the last stable version to check if we really needed alpha and everything seems to work fine! So I thought we would prefer to have the stable version of storybook

Wdyt? We can go back to alpha

Reply:
for me stable is good if it works 👍